scrapping/Day.py

```diff
@@ -0,0 +1,14 @@
+
+class Day:
```

Comment:
Pls pune astea in engleza, si si restul denumirilor :D

Reply:
Dacate refereai la numele zilelor, rezolvat